2-Draft-Accepted/RFCNNNN-EnvironmentVariable-Cmdlets.md

```diff
@@ -0,0 +1,386 @@
+---
+RFC:
+Author: Charu Bassi
+Status: Draft
+SupercededBy: n/a
+Version: 0.1
+Area: Cmdlet
+Comments Due: 5/10/2017
+---
+
+# *-EnvironmentVariable Cmdlets
+
+Add the new cmdlets to PowerShell to get\set\create the environment variables for machine\process\user.
+
+Currently users are forced to use the .Net api's to retreive\set the value of environment variables :
```

Comment:
Suggest "... retrieve\set the value of environment variables for the user and machine scopes"
The environment provider works fine for setting process scoped environment variables.
```
$ENV:foo = "bar"
```